contracts/WildcardsQV.sol

```diff
@@ -247,23 +274,35 @@ contract WildcardsQV is Initializable {
 
         // Collect patronage on the WildCard
         wildCardSteward._collectPatronage(dragonCardId);
+
+        uint256 amountRaisedInIteration = wildCardSteward.benefactorFunds(
+            _thisAddress
+        );
```

Comment:
Okay this is important. We are first collecting the Patronage, then we are checking the BenfactorFunds, how is the BenefactorFunds calculated? Is it only what is expressly been collected and available, or does it take into account uncollected patroange? If the latter is the case, then amountRaisedInIteration will be > than what we have collected, if only slightly, but break the contract.

Reply:
In the original code (wildcards) this is that mapping; when the token is has 'collect patronage' run on it that mapping displays how much the 'benefactor' can withdraw.

Reply:
> Is it only what is expressly been collected and available, or does it take into account uncollected patroange?

The previous line (line 277) collects the patronage, so there is no problem I can see right?

Reply:
Got it makes sense